sbnobj/Common/POTAccounting/classes_def.xml

```diff
@@ -11,7 +11,9 @@
    <version ClassVersion="11" checksum="4062206176"/>
    <version ClassVersion="10" checksum="3057205612"/>
   </class>
-  <class name="sbn::EXTCountInfo" ClassVersion="11">
+  <class name="sbn::EXTCountInfo" ClassVersion="13">
+   <version ClassVersion="13" checksum="90106564"/>
+   <version ClassVersion="12" checksum="3544499454"/>
```

Comment:
That would be:
```suggestion
  <class name="sbn::EXTCountInfo" ClassVersion="12">
   <version ClassVersion="12" checksum="90106564"/>
```
Rule of thumb is that a pull request should add at most one version per class.